src/MetricsBuilder.php

```diff
@@ -246,6 +251,20 @@ private function appendCredentialsMetric(
         }
     }
 
+    private function appendRequestChecksumCalculationMetric(
```

Comment:
We should probably have assertions on the corresponding test class for this metrics

Reply:
That method is implicitly covered [here](https://github.com/aws/aws-sdk-php/pull/3065/files#diff-8aabda90e2b5d80c8054b3f415b3478853e44ee9d1c48824bc29465837e91f84R1566), however, if you think we need individual tests for those methods I can do it.